en/core-libraries/components/pagination.rst

```diff
@@ -90,6 +90,11 @@ array after the model you wish to configure::
 The values of the ``Post`` and ``Author`` keys could contain all the properties
 that a model/key less ``$paginate`` array could.
 
+.. note::
+    The ``$paginate`` variable is not automatically passed to the :php:class:`PaginatorComponent`'s settings. You must explicitly set the :php:class:`PaginatorComponent`'s settings using the following line::
+
+        $this->Paginator->settings = $this->paginate;
```

Comment:
You should not mix those two.
Either you use $this->paginate and $this->paginate() or you use the helper and the helper property.
Mixing those always got you in trouble in the past, and it will also do so from here on.
So I think you should rather point out the consistent approach of those two ways then mixing them up.